migrate_async_to_gr.yml

```diff
@@ -0,0 +1,78 @@
+#
+# This playbook migrates the primary-replicas setup generated for mysql1,mysql2,mysql3 servers into a Group Replication Cluster
+#
+
+## Settings for ProxySQL tutorial
```

Comment:
??

Reply:
ProxySQL tutorial?

Reply:
Right. We have a version of the proxysql tutorial lab on which people set up proxysql to work against a GR backend. Until now you have to deploy mysql1, mysql2, mysql3 and manually convert those to GR for people to take the training. With this playbook you can apply it after the hosts.yml and you have a GR-enabled env ready to go.